src/ReachSets/reach.jl

```diff
@@ -211,6 +214,130 @@ function reach(S::AbstractSystem,
     available_algorithms[algorithm_backend]["func"](args...)
     tocc()
 
+
     # return the result
     return res
 end
+
+
+
+"""
+    reach(S, N; [algorithm], [ε_init], [set_type_init], [ε_iter],
+          [set_type_iter], [assume_sparse], [assume_homogeneous],
+          [numeric_type], [lazy_X0], [kwargs]...)
+
+Interface to reachability algorithms for an LTI system.
+
+### Input
+
+- `HS`                 -- Hybrid System
+- `N`                  -- number of computed sets
+- `algorithm`          -- (optional, default: `"explicit"`), reachability
+                          algorithm backend; see `available_algorithms` for all
+                          admissible options
+- `ε_init`             -- (optional, default: `Inf`) error bound for the
+                          approximation of the initial states (during
+                          decomposition)
+- `set_type_init`      -- (optional, default: `Hyperrectangle`) set type for the
+                          approximation of the initial states (during
+                          decomposition)
+- `ε_iter`             -- (optional, default: `Inf`) error bound for the
+                          approximation of the states ``X_k``, ``k>0``
+- `set_type_iter`      -- (optional, default: `Hyperrectangle`) set type for the
+                          approximation of the states ``X_k``, ``k>0``
+- `assume_sparse`      -- (optional, default: `true`) if true, it is assumed
+                          that the coefficients matrix (exponential) is sparse;
+                          otherwise, it is transformed to a full matrix
+- `assume_homogeneous` -- (optional, default: `false`) if true, it is assumed
+                          that the system has no input (linear system), and in
+                          case it has one, the input is ignored
+- `numeric_type`       -- (optional, default: `Float64`) numeric type of the
+                          resulting set
+- `lazy_X0`            -- (optional, default: `false`) if true, transform the
+                          set of initial states to the caretsian product of
+                          two-dimensional polygons; otherwise, the given input,
+                          as a lazy set, is passed to the backend
+- `kwargs`             -- (optional) additional arguments that are passed to the
+                          backend
+
+### Notes
+
+A dictionary with available algorithms is available via
+`Reachability.available_algorithms`.
+"""
+function d_reach(HS::HybridSystem, # Does AbstractSystem a superset of HybridSystem?
+               N::Int;
+               X0::BallInf; # TODO is it correct to way to specify initial set for HS?
```

Comment:
No, just write `X0::LazySet`, meaning that you accept a general set type.

Reply:
Fixed